java/arcs/android/devtools/DevToolsService.kt

```diff
@@ -27,15 +29,29 @@ class DevToolsService : Service() {
 
     private val coroutineContext = Dispatchers.Default + CoroutineName("DevtoolsService")
     private val scope = CoroutineScope(coroutineContext)
-    private val binder = DevToolsBinder(scope)
+    private lateinit var binder: DevToolsBinder // = DevToolsBinder(scope)
+    private var devToolsServer = DevWebServerImpl
+
+    override fun onCreate() {
+        binder = DevToolsBinder(scope, devToolsServer as DevWebServer)
+        scope.launch {
+            suspendCancellableCoroutine { cont ->
+                cont.invokeOnCancellation {
+                    devToolsServer.close()
+                }
+                devToolsServer.startIfNeeded()
```

Comment:
Can `start` be used now, since there's only one, and it'll be started only once anyway? (onCreate is only called once)